homeassistant/components/vallox/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "vallox",
+  "name": "Vallox",
+  "documentation": "https://www.home-assistant.io/components/vallox",
+  "requirements": [
+    "vallox-websocket-api==1.5.1",
+    "numpy==1.16.3",
+    "websocket-client==0.54.0"
```

Comment:
Why do we need to include the websocket client as a requirement? I would think that the vallox websocket api library would depend on the websocket client library.

Reply:
Why include numpy? We don't seem to use that.

Reply:
vallox-websockets-api lib uses that. But it should be pulled as lib dependency then.